src/components/ButtonPanel.js

```diff
@@ -1,24 +1,41 @@
+import PropTypes from 'prop-types';
 import Button from './Button';
 import '../styles/ButtonPanel.css';
 
-const ButtonPanel = () => (
+const Panel = ({ onClick, symbols }) => symbols.map((symbol, index) => (
+  <Button
+    key={(symbol + index).toString()}
+    buttonName={symbol}
+    onClick={onClick}
+  />
+));
+
+const ButtonPanel = ({ onClick }) => (
```

Comment:
[Required] Hello @Maryjanee, I forgot to mention that you need to create functions in the ButtonPanel and Button component as mentioned in the requirement. Even though your app is running smoothly, it is a hard requirement.  Please reference the [curriculum](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376019/)